patches/server/0913-Add-Structure-check-API.patch

```diff
@@ -17,7 +17,7 @@ index 612dc787880e58e8325a658a63f9fe7536d0860c..ebb119ab9f5a8ae580e54cb3c102cd86
 +    public boolean hasStructureAt(final io.papermc.paper.math.Position position, final Structure structure) {
 +        net.minecraft.world.level.levelgen.structure.Structure vanillaStructure = this.world.registryAccess()
 +            .registryOrThrow(net.minecraft.core.registries.Registries.STRUCTURE)
-+            .getHolder(CraftNamespacedKey.toMinecraft(structure.getStructureType().getKey()))
++            .getHolder(CraftNamespacedKey.toMinecraft(structure.getKey()))
 +            .orElseThrow()
 +            .value();
```

Comment:
This should really just be replaced with `CraftStructure.bukkitToMinecraft()` now I think. Upstream added a bunch of utility methods in the "Craft" classes to convert between API and nms types.

Reply:
Should I put it back to a one liner?

Reply:
I mean it can be. Or just assign the `vanillaStructure` local var to `CraftStructure.bukkitToMinecraft(structure);`.